### ISSUE TITLE
Failure with dev ggplot2

### ISSUE THREAD

**Issue by hadley** [tengfei/ggbio#101]
It looks like ggbio causes some downstream failures with dev ggplot2 due to the switch to tidy evaluation: https://github.com/tidyverse/ggplot2/issues/2610#issuecomment-389579542

Please let us know if you need help figuring out a fix!

**Comment by lawremi** [comment]
It looks like the tilde is being used as a sort of sentinel value, e.g., `ymax` can be a quosure holding the symbol `~`.  Is there some convention around this in rlang?

**Comment by hadley** [comment]
Hmmmm there shouldn’t be. What does the aes() call look like?

**Comment by lawremi** [comment]
Nevermind, turns out something in ggbio had to be updated.

**Comment by greysabarrios** [comment]
Hello everyone. I'm new to bioinformatics, and recently I started using R and ggbio to graph my genomic data. I made a script that had run well until yesterday, when I updated packages for the new version of R that came out (3.5.1).

Now, I try to run my script and the error you mention appears. This is the error:
`Error in min(c(x, xmin), na.rm = TRUE) : 
  invalid 'type' (list) of argument`

I'm desperate, because I have to make another graphic ... and I do not know how to solve that error.

How can I solve it?

This is my script:
`ilename <- "cromosoma.txt"
datos1 <- read.delim2 (filename, sep = "\t", header = TRUE )
head (datos1)

library (GenomicRanges)
library (ggbio)

plot <- GRanges(datos1, seqnames = c(datos1$Cromosoma), strand = NULL, 
      ranges = IRanges(start = c (datos1$Inicio), end = c(datos1$Fin)))
      seqinfo = (plot)

head (plot)
      
seqlengths(plot) <-  c (37320000, 41240000, 36360000, 31930000, 39370000,
                    26300000,  21610000, 19590000,  38630000,  21820000)

plot <- renameSeqlevels (plot, paste0("Chr", c(1:10)))

autoplot(plot, layout = "karyogram", aes(color = Tipo, fill = Tipo))`

Thank you very much
